applications/tari_dan_wallet_web_ui/src/utils/json_rpc.tsx

```diff
@@ -69,7 +69,7 @@ export async function jsonRpc(method: string, params: any = null) {
     if (token === null) {
       let auth_response = await internalJsonRpc("auth.request", null, [["Admin"], null]);
       let auth_token = auth_response["auth_token"];
-      let accept_response = await internalJsonRpc("auth.accept", null, [auth_token, "AdminToken"]);
+      let accept_response = await internalJsonRpc("auth.accept", null, [auth_token, auth_token]);
```

Comment:
The 2nd parameter of auth.accept is the user friendly name that will be shown in the dan wallet ui. Anyway, this will be all removed in the upcoming changes with just one admin token per wallet.